### PR TITLE
Remove interactions on disabled card

### PR DESCRIPTION
**Pull request by robinandeer**
## What?

Group all disabled-styles.
Add Card stories to Storybook.

## Why?

Since the card is getting some different state, I thought it might be worth adding it to Storybook. Let me know if you think it makes sense also!

https://user-images.githubusercontent.com/1220232/117307479-e66ecc80-ae80-11eb-8ddc-541c1b3d7a84.mov

### CODE REVIEW

File: src/client/pages/Landing/components/Card.tsx [hunk 1]
```diff
@@ -46,14 +44,25 @@ const CardComponent = styled.div<{ disabled?: boolean }>`
...
+
+          :hover {
+            transform: translateY(-6px);
+            color: ${colorsV3.gray900};
+
+            @media (hover: none) {
+              transform: none;
+            }
+          }
+        `}
```
Comment (gustaveen) [inline src/client/pages/Landing/components/Card.tsx:65]: 👌

File: src/client/pages/Landing/components/Card.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { MemoryRouter } from 'react-router-dom'
+import { Card } from './Card'
+
+export default {
+  title: 'Landing Card',
```
Comment (gustaveen) [inline src/client/pages/Landing/components/Card.stories.tsx:6]: Maybe group it as a component under Landing? :) 
```suggestion
  title: 'Landing/Card',
```

**Review by gustaveen — DISMISSED** [review]
Awesome! Extra points for adding it to Storybook 🎉

**Review by styrelius — APPROVED** [review]
💅

**Pull request merged**